tensorrt_llm/_torch/visual_gen/mapping.py

```
@@ -138,11 +138,13 @@ def build_mesh(self):
         if cls.device_mesh is not None:
             return
 
-        shape = tuple(self._dim_sizes[d] for d in self._dim_names)
+        dims = ("pp",) + (self._dim_names) + ("cp",)
```

Ruff (0.15.12)

[warning] Consider `("pp", *self._dim_names, "cp")` instead of concatenation

Replace with `("pp", *self._dim_names, "cp")`

(RUF005)